### ISSUE TITLE
  uint32_t hidddenSectors should be hiddenSectors?

### ISSUE THREAD

**Issue by ledentist-0x01** [greiman/SdFat#197]
in  SdFat/src/FatLib/FatStructs.h, Line 197 is stated: uint32_t hidddenSectors, looks like typo.

**Comment by greiman** [comment]
Yes.  I will change it if SdFat is ever updated.  It is never used since it dates back to old MSDOS days.

I don't plan any new development.  SdFat-beta will eventual replace this version of SdFat.
